### PR TITLE
Use only groups in RBACs

### PR DESCRIPTION
**Pull request by hongkailiu**
Those are existing violations in the release repo.

/cc @openshift/test-platform

### CODE REVIEW

**Comment by petr-muller** [comment]
We;ll need to coordnate this really well to not break people. Basically we need to make sure all equivalents exist as a Rover groups first.

**Comment by hongkailiu** [comment]
> We;ll need to coordnate this really well to not break people. Basically we need to make sure all equivalents exist as a Rover groups first.

Agreed. We will discuss the steps offline.

File: clusters/app.ci/assets/admin_ci-monitoring-view_rbac.yaml [hunk 1]
```diff
@@ -1,12 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: ci-monitoring-view
```
Comment (hongkailiu) [inline clusters/app.ci/assets/admin_ci-monitoring-view_rbac.yaml:4]: After merge:

```console
oc delete ClusterRoleBinding ci-monitoring-view --context app.ci
```

This one is replaced by `clusters/build-clusters/common/monitoring_viewer.yaml`.

File: clusters/build-clusters/common/monitoring_viewer.yaml [hunk 2]
```diff
@@ -9,12 +9,4 @@ roleRef:
...
 - kind: Group
-  name: cluster-monitoring-viewer
----
-apiVersion: user.openshift.io/v1
-kind: Group
-metadata:
-  name: cluster-monitoring-viewer
-users:
-- YuviGold
-- wking
```
Comment (hongkailiu) [inline clusters/build-clusters/common/monitoring_viewer.yaml:20]: wking is added into the rover group.

I cannot see YuviGold in github openshift org: https://github.com/orgs/openshift/people
Either in rover.

**Comment by hongkailiu** [comment]
/hold

Need to announce it first.

**Pull request merged**